From 263449c0667a03f366c801cf240e64ca0b464d47 Mon Sep 17 00:00:00 2001
From: Alexandru Gagniuc <mr.nuke.me@gmail.com>
Date: Sat, 30 Mar 2024 20:03:30 -0500
Subject: [PATCH] arm64: dts: qcom: ipq9574: Use 'usb-phy' for node names

The devicetree spec allows node names of "usb-phy". So be more
specific for the USB PHYs, and name the nodes "usb-phy" instead of
just "phy".

Signed-off-by: Alexandru Gagniuc <mr.nuke.me@gmail.com>
---
 arch/arm64/boot/dts/qcom/ipq9574.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm64/boot/dts/qcom/ipq9574.dtsi
+++ b/arch/arm64/boot/dts/qcom/ipq9574.dtsi
@@ -722,7 +722,7 @@
 			status = "disabled";
 		};
 
-		usb_0_qusbphy: phy@7b000 {
+		usb_0_qusbphy: usb-phy@7b000 {
 			compatible = "qcom,ipq9574-qusb2-phy";
 			reg = <0x0007b000 0x180>;
 			#phy-cells = <0>;
@@ -736,7 +736,7 @@
 			status = "disabled";
 		};
 
-		usb_0_qmpphy: phy@7d000 {
+		usb_0_qmpphy: usb-phy@7d000 {
 			compatible = "qcom,ipq9574-qmp-usb3-phy";
 			reg = <0x0007d000 0xa00>;
 			#phy-cells = <0>;
